pkg/api/const.go

```diff
@@ -61,8 +61,10 @@ const (
 	Kubernetes162 OrchestratorVersion = "1.6.2"
 	// Kubernetes166 is the string constant for Kubernetes 1.6.6
 	Kubernetes166 OrchestratorVersion = "1.6.6"
+	// Kubernetes166 is the string constant for Kubernetes 1.6.6
+	Kubernetes170 OrchestratorVersion = "1.7.0"
 	// KubernetesLatest is the string constant for latest Kubernetes version
-	KubernetesLatest OrchestratorVersion = Kubernetes166
+	KubernetesLatest OrchestratorVersion = Kubernetes170
```

Comment:
I was thinking that when I was doing the patch.
I believe this only occurs when you do not specify orchestratorVersion in your json.
Will update to reflect your comments.